.travis.yml

```diff
@@ -81,7 +81,7 @@ script:
   - mkdir build && cd build
   - $CMAKE_CONFIGURE cmake $CMAKE_ARGS $CMAKE_EXTRA ..
   - make -j2
-  - ctest -j2 --output-on-failure
+  - travis_wait 30 ctest -j2 --output-on-failure
```

Comment:
why is this needed ?

Reply:
Travis during the Valgrind test is timing out because no output is received for 10 minutes since the PDB and GRO tests take longer than this to complete (and they take up both cores). This is to stop Travis from killing the job. I'm happy to look into other fixes for this. For example, is `--track-origins=yes` needed? I know this can increase Valgrind runtime significantly.

On my machine, both PDB and GRO tests complete in seconds, not 10 minutes - so I'm not sure why Travis takes so long.

Reply:
Ok! I did not know what the travis_wait command did :smile: 

`--track-origins=yes` might not be needed here, and I have a local branch to improve this a bit too by not running the longest test with Travis + valgrind. I'll push it as soon as possible.

> On my machine, both PDB and GRO tests complete in seconds, not 10 minutes - so I'm not sure why Travis takes so long.

By default, the tests are not run under valgrind. And Travis machines are not particularly powerful ones.

Reply:
I meant with `-DVALGRIND=ON` used on my local machine :smiley: 

I think the issue is that Travis does not provide powerful machines, but its free so I can't complain :roll_eyes: .

`travis_wait 30` seems to have solved the issue (looking at the timing I could have used a number smaller than 30, I just wanted to be sure).